js/app.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+var dropMenu = document.getElementById('dropdown');
+var userForm = document.getElementById('user-form');
+var productForm = document.getElementById('product-container');
+var formSub = document.getElementById('submit');
+Product.all = [];
+
+//Need to check local storage if user exits/ has products in cart, etc.
+function User(name, street, city, state, zip, phone) {
+  this.name = name;
+  this.street = street;
+  this.city = city;
+  this.state = state;
+  this.zip = zip;
+  this.phone = phone;
+
+  //Array of products in the user cart
+  this.cart = [];
+}
+
+User.prototype.addProduct = function(product, quantity) {
+  for(var i = 0; i < quantity; i++) {
+    this.cart.push(product);
+  }
+  console.log('Cart: ' + this.cart);
+};
+
+function Product(name, filePath) {
+  this.name = name;
+  this.filePath = filePath;
+  this.index = Product.all.length;
+
+  //Create dropdown list option element
+  var opEl = document.createElement('option');
+  opEl.value = Product.all.length;
+  opEl.textContent = name;
+  dropMenu.appendChild(opEl);
```

Comment:
This should probably be in a method rather than the constructor, just because it is unrelated to actually constructing the object. You can then call that method in the constructor.